src/app/parts-editor/components/dialog-database/dialog-database.component.html

```diff
@@ -0,0 +1,20 @@
+<h1 i18n>Parts missing from database</h1>
+
+<form [formGroup]="missingPartsForm" (ngSubmit)="onSubmit()">
+  <div formArrayName="updatedParts">
+    <div *ngFor="let part of updatedParts().controls; let partIndex = index">
+      <div [formGroupName]="partIndex">
+        <input type="text" readonly formControlName="rowIndex" style="width: auto" />
+        <span i18n>Report number :</span>
```

Comment:
There is dedicated label html tag for labels. Do not use ordinary spans for that. Also it would be much better to use a table layout as labels are the same for each row